runtime/src/main/java/org/corfudb/runtime/collections/Table.java

```diff
@@ -322,6 +324,62 @@ public void preCommitCallback(TokenResponse tokenResponse) {
         return keyOfQueueEntry;
     }
 
+    /**
+     * Apply a Corfu SMREntry directly to a stream. This can be used for replaying the mutations
+     * directly into the underlying stream bypassing the object layer entirely.
+     *
+     * @param e the element to update
+     * @param streamTags  - stream tags associated to the given stream id
+     * @param corfuStore CorfuStore that gets the runtime for the serializer.
+     * @throws IllegalArgumentException if some property of the specified
```

Comment:
prevents it from being added to the queue

Reply:
Changed.